response_operations_ui/templates/collection_exercise/ce-handle-errors.html

```diff
@@ -1,8 +1,8 @@
 {#- Display processing or error panel if validation failed #}
 {% if processing %}
-    {% call panel.render(id='processing-info', class='u-mb-s') %}Processing collection exercise – <a id="a-processing-refresh" href="">refresh to see progress</a>{% endcall %}
+    {% call panel(id='processing-info', class='u-mb-s', custom_attribs={cheese: 'wensleydale', hotdog: 'hot'}) %}Processing collection exercise – <a id="a-processing-refresh" href="">refresh to see progress</a>{% endcall %}
```

Comment:
I like cheese and hot-dogs, but not sure they're meant to be here

Reply:
👀

Reply:
This seems like a fair point